plugins/poll/config/settings.yml

```diff
@@ -30,6 +30,10 @@ plugins:
     default: -16
     min: -9999
     client: true
+  poll_export_data_explorer_query_id_irv:
+    default: 0
```

Comment:
Is `0` a good default? 🤔

Reply:
this was part placeholder, but I might leverage that 0 to prevent the option being offered unless (at a later stage) an extract query is added.

Reply:
@ZogStriP 

Raised PR here for new export query: https://github.com/discourse/discourse-data-explorer/pull/294

Updated code for plugin accordingly: https://github.com/discourse/discourse/pull/27155/commits/9f94fe4aa0c9135ec60b428fd28aa39e2b1eb729